src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -1430,6 +1485,33 @@ public static int MethodWithDefaultParameter(int x, int y = 10)
             return x + y;
         }
 
+        public static int MethodWithProgressParameter(IProgress<int> p)
```

Comment:
>MethodWithProgressParameter(IProgress<int> p) [](start = 26, length = 45)

We should create a new nested class (sibling to the Server class) that defines just one method which takes a `Progress<int>` parameter, and then add a test that verifies that this _fails_ when added as a target object in some acceptable way, since we only support server methods that accept `IProgress<T>` and not the concrete type `Progress<T>`.